compiler/crates/relay-compiler/src/artifact_content/content.rs

```diff
@@ -765,8 +765,8 @@ fn generate_disable_lint_section(language: &TypegenLanguage) -> Result<GenericSe
     let mut section = GenericSection::default();
     match language {
         TypegenLanguage::TypeScript => {
-            writeln!(section, "/* tslint:disable */")?;
             writeln!(section, "/* eslint-disable */")?;
+            writeln!(section, "/* tslint:disable */")?;
```

Comment:
tslint has been deprecated since 2019. Better to just remove it imo

Reply:
done! 😄